lib/src/util/event_handler_error_catcher.dart

```diff
@@ -0,0 +1,45 @@
+// @dart = 2.7
+
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:html';
+
+import 'package:react/react_client/react_interop.dart' show throwErrorFromJS;
+
+/// INTENDED FOR INTERNAL USE IN REACT_TESTING_LIBRARY ONLY.
+///
+/// Event Handlers triggered by a UserEvent or fireEvent may throw errors
```

Comment:
This could use a dartdoc-style opening sentence.
```suggestion
/// Calls [cb] and throws if any `window.onError` error that occur during its execution.
///
/// Event Handlers triggered by a UserEvent or fireEvent may throw errors
```